posters.json

```diff
@@ -16810,7 +16810,7 @@
    "presenter": "David Kennedy",
    "categories": "Neuroinformatics and Data Sharing<br>Informatics Other",
    "videochat": "<a href=\"https://meet.jit.si/ohbm2020-1976\" target=\"_ohbm2020-1976\">jitsi:ohbm2020-1976</a>",
-   "pdf": ""
+   "pdf": "https://doi.org/10.5281/zenodo.3906257"
```

Comment:
should go into posters-overrides.json not into posters.json!

Reply:
Ah, who knew. No instructions provided for the un-initiated...  Will try again...

Reply:
hm, PR request template should have come up with instructions... I guess it might have not -- need to check